exchange/utils.go

```diff
@@ -94,7 +95,12 @@ func cleanOpenRTBRequests(ctx context.Context,
 	privacyLabels.COPPAEnforced = privacyEnforcement.COPPA
 	privacyLabels.LMTEnforced = lmtEnforcer.ShouldEnforce(unknownBidder)
 
-	if gdpr == 1 {
+	GDPREnabled := accountLevelGDPREnabled(account, labels.RType)
+	if GDPREnabled == nil {
+		GDPREnabled = hostLevelGDPREnabled(privacyConfig)
+	}
+
+	if gdpr == 1 && *GDPREnabled {
```

Comment:
This made me scratch my head for what the `if` statement would do if the `GDPREnabled` is nil. Segfault? That's not possible with the code path, so perhaps a bit of a refactor can eliminate that uncertainty. Consider moving the host level check into the helper func and have the func return a bool instead of a bool pointer.